include/swift/SIL/DebugUtils.h

```diff
@@ -437,16 +437,16 @@ struct DebugVarCarryingInst : VarDeclCarryingInst {
 
   Kind getKind() const { return Kind(VarDeclCarryingInst::getKind()); }
 
-  std::optional<SILDebugVariable> getVarInfo() const {
+  std::optional<SILDebugVariable> getVarInfo(bool complete = true) const {
```

Comment:
Can you document what the complete flag does and under what circumstanced we would not want the default value here?